mlir/include/mlir/Dialect/Rock/utility/loweringUtils.h

```diff
@@ -22,6 +22,76 @@ namespace rock {
 struct ConvolutionDims;
 struct GemmSize;
 
+// This is a helper data structure
+// to server as a return type for
+// view creating utilities.
+struct GPUViews {
+  // This is a [gridIdx0, ... , gridIdxN, tid, iter] to K x D view
+  // This is useful to define a global tile once loaded into regs.
+  ArrayAttr gridwiseView;
+  // This is a [tid, iter] to flat K x D view
+  // Useful to define a blockwise tile once loaded into regs.
+  ArrayAttr blockwiseView;
+  // This is a [iter] to flat K x D view
+  // Useful to define a threadwise tile once loaded into regs.
```

Comment:
Somewhere, either in this file or elsewhere, define what this function is and how you're meant to use it.